### PR TITLE
Adds Pools to Metastation

### PR DESCRIPTION
**Pull request by totallynotkatfood**
## About The Pull Request

Adds the pool, and also the pool noodle.

## Why It's Good For The Game

There's a grill in the pool room, and the fitness room isn't that interesting.

## Changelog
:cl:
add: Added a pool to Meta
add: Added pool noodles
soundadd: Added splashing sounds and that crap
imageadd: Added icons for the pool
code: Pool coode
/:cl:

### CODE REVIEW

**Comment by TheFakeElon** [comment]
I don't think you made enough changelog entries

**Comment by MCterra10** [comment]
this map change is going to be horrific

**Comment by MCterra10** [comment]
to handle that is. oh god I can just imagine the conflicts

**Comment by MCterra10** [comment]
what I'd recommend you do is make a copy of metastation, call it austation, and just basically make a new map.

**Comment by MCterra10** [comment]
I'll put that in rotation

**Comment by totallynotkatfood** [comment]
travis broke smh

File: beestation.dme [hunk 1]
```diff
@@ -84,6 +84,7 @@
 #include "code\__DEFINES\pinpointers.dm"
 #include "code\__DEFINES\pipe_construction.dm"
 #include "code\__DEFINES\plumbing.dm"
+#include "code\__DEFINES\pool.dm"
```
[the remark this answers is not in the document]
Comment (totallynotkatfood) [reply beestation.dme:87]: I can’t modularize it, it contains a global list, everything will break

[the remark this answers is not in the document]
Comment (MCterra10) [reply beestation.dme:87]: just include it in includes.dm instead of define_includes.dm lol.

[the remark this answers is not in the document]
Comment (MCterra10) [reply beestation.dme:87]: define includes is for stuff that has to be before everything else

[the remark this answers is not in the document]
Comment (MCterra10) [reply beestation.dme:87]: actually, leave the pool defines where they are, put those in define_includes.dm.
*BUT* put the global list in a new file in and include that in includes.dm

**Comment by MCterra10** [comment]
conflicts

File: beestation.dme (hunk 1 above)
[the remark this answers is not in the document]
Comment (MCterra10) [reply beestation.dme:85]: resolving this myself, one sec

File: austation/code/modules/pool/pool_main.dm [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+						H.Knockdown(200) // This should hurt. And it does.
+						playsound(src, 'sound/effects/woodhit.ogg', 60, TRUE, 1)
+						playsound(src, 'austation/sound/misc/crack.ogg', 100, TRUE)
+				else
+					H.visible_message("<span class='danger'>[H] falls in the drained pool, but had an helmet!</span>",
+										"<span class='userdanger'>You fall in the drained pool, but you had an helmet!</span>")
+					H.Knockdown(40)
+					playsound(src, 'sound/effects/woodhit.ogg', 60, TRUE, 1)
+		else if(filled)
+			victim.adjustStaminaLoss(1)
```
Comment (MCterra10) [inline austation/code/modules/pool/pool_main.dm:153]: excuse me, no thanks

Comment (MCterra10) [reply austation/code/modules/pool/pool_main.dm:153]: remove the stamloss shit.

**Comment by MCterra10** [comment]
updating config now

**Pull request merged**